lib/core/src/test/java/io/atlasmap/core/DefaultAtlasContextTest.java

```diff
@@ -61,6 +61,31 @@
 
 public class DefaultAtlasContextTest extends BaseDefaultAtlasContextTest {
 
+    @Test
+    public void testMappingsVersion() throws AtlasException {
```

Comment:
I suspect this doesn't trigger version check? would need to invoke `context.init()`...
or invoke version check also from `context.processValidation()` and put the warn level `Audit`

Reply:
I went the nvoke version check also from context.processValidation() and put the warn level Audit way.